src/SmartComponents/Inventory/applications/vulnerabilities/DataMapper.js

```diff
@@ -18,13 +10,21 @@ export function createCveListBySystem({ isLoading, ...rest }) {
             data: data.map(row => ({
                 id: row.id,
                 cells: [
-                    <span key={ row.id }>
-                        <SecurityIcon size="md" color={ colorToImpact[row.attributes.impact] } />
-                    </span>,
+                    <Shield
+                        impact={ row.attributes.impact }
+                        hasTooltip={ true }
+                        tooltipPosition={ 'right' }
+                        key={ row.id.toString() }
+                    />,
                     row.attributes.synopsis,
-                    <span key={ `title-${row.id}` } title={ row.attributes.description }>{ row.attributes.description.substr(0, 199) }...</span>,
+                    <LongTextTooltip
+                        content={ row.attributes.description }
+                        tooltipMaxWidth={ '50vw' }
```

Comment:
Why are we styling using the viewport?

Reply:
What other solution do you suggest? This is the default size for the `LongTextTooltip` component.